reboot-via-dsme.sh

```diff
@@ -0,0 +1,15 @@
+# Define shell functions for interactive shells only
+[ -z "$PS1" ] && return
```

Comment:
The same $PS1 check is used in /etc/bashrc that sources the script, the most voted solution is one liner of bashisms... I guess this should work?

    case "$-" in *i*) ;; *) return ;; esac